client/app/models/project.js

```diff
@@ -112,8 +112,8 @@ export default class ProjectModel extends Model {
 
   get eisPackages() {
     const eisPackages = [
-      ...this.deisPackages,
       ...this.feisPackages,
+      ...this.deisPackages,
```

Comment:
Weird, this change should already be in develop

Reply:
Why is this diff still rendering? Very strange...

Reply:
Please check again